cultivation/scripts/running/parse_run_files.py

```
@@ -29,141 +60,354 @@ def parse_fit_with_fitdecode(file_path):
     # --- Integrate metrics if GPX ---
-    if hasattr(df, '_is_gpx_metrics') or 'pace_sec_km' in df.columns:
+    if 'pace_sec_km' in df.columns:
         try:
             from metrics import run_metrics
             metrics = run_metrics(df, threshold_hr=175, resting_hr=50)
             summary.update(metrics)
         except Exception as e:
-            print(f"[metrics] Could not compute advanced metrics: {e}")
-
-    print(f"\nSummary for {label}:")
+            if VERBOSE: print(f"[metrics] Could not compute advanced metrics: {e}")
```

Ruff (0.8.2)

Multiple statements on one line (colon)

(E701)